examples/faq/BERT-incorporate-email-context.py

```diff
@@ -0,0 +1,65 @@
+import simpletransformers
```

Comment:
After the paper submission and things slow down, I'm thinking it could be more intuitive to pull out everything from `simpletransformers/examples/faq` into `ds-faq-pilot` or a new repo with FAQ/content rec experiments.  

Goal: keep `simpletransformers` as the library with generic models and logic and leverage in task specific repos such that when we have a pattern that scales when we have more and more models, say 10 models using transformers.